### PR TITLE
PCHR-2358: Add user guide link + modified footer in ssp pages

### PR DESCRIPTION
**Pull request by bmnepali**
## Overview
- A new sub-menu in account menu in SSP. 
- Modify footer for SSP pages

## After
Displays more information in footer
screen shot 2017-07-12 at 8 02 02 pm

## After
Displays only required information in footer.
screen shot 2017-07-12 at 7 51 23 pm

## Technical Details:
1. Modify "civihr_default_theme/templates/page/page.tpl.php" for sub menu
```html
<ul class="chr_header__sub-menu">
   ...
   <li><?php print $user_guide_link; ?></li>
</ul>
```
2. Modify "civihr_default_theme/templates/page/page.tpl.php" file for footer.
```html
<div class="text-center">
    Powered by CiviHR <?php print get_civihr_version(); ?>.
    CiviHR is openly available under the <a href="http://www.gnu.org/licenses/agpl-3.0.html">GNU 
    AGPL License</a> and can be downloaded from
    <a target="_blank" href="https://civihr.org">Project website</a>&nbsp;.
    <div class="footer-logo">
    <i class="icon-logo-full"></i>
</div>
```

### CODE REVIEW

**Comment by mickadoo** [comment]
Typo in description "file for tooter"

**Review by mickadoo — APPROVED** [review]
Looks good :sunflower:

**Comment by bmnepali** [comment]
Updated PR description!

File: civihr_default_theme/templates/page/page.tpl.php [hunk 1]
```diff
@@ -149,12 +150,8 @@
     <div class="container">
       <div class="text-center">
         Powered by CiviHR <?php print get_civihr_version(); ?>.
-        CiviHR is openly available under the <a href="http://www.gnu.org/licenses/agpl-3.0.html ">GNU AGPL License</a>.
-        <br />
-        <a target="_blank" href="http://civihr-documentation.readthedocs.io/en/latest/">User Guide</a>&nbsp;
-        <a target="_blank" href="https://github.com/civicrm/civihr">Download CiviHR</a>&nbsp;
-        <a target="_blank" href="https://civihr.atlassian.net/wiki/display/CIV/Welcome ">View Wiki page</a>&nbsp;
-        <a target="_blank" href="https://civihr.org">Project website</a>
+        CiviHR is openly available under the <a href="http://www.gnu.org/licenses/agpl-3.0.html">GNU AGPL License</a> and can be downloaded from
```
Comment (mickadoo) [inline civihr_default_theme/templates/page/page.tpl.php]: Maybe you should check with Mike, but "downloaded from Project Website" seems a bit strange without a "the"

**Review by mickadoo — COMMENTED** [review]
Just a question on something I missed first time around

File: civihr_default_theme/templates/page/page.tpl.php [hunk 2]
```diff
@@ -149,12 +150,7 @@
     <div class="container">
       <div class="text-center">
         Powered by CiviHR <?php print get_civihr_version(); ?>.
-        CiviHR is openly available under the <a href="http://www.gnu.org/licenses/agpl-3.0.html ">GNU AGPL License</a>.
-        <br />
-        <a target="_blank" href="http://civihr-documentation.readthedocs.io/en/latest/">User Guide</a>&nbsp;
-        <a target="_blank" href="https://github.com/civicrm/civihr">Download CiviHR</a>&nbsp;
-        <a target="_blank" href="https://civihr.atlassian.net/wiki/display/CIV/Welcome ">View Wiki page</a>&nbsp;
-        <a target="_blank" href="https://civihr.org">Project website</a>
+        CiviHR is openly available under the <a href="http://www.gnu.org/licenses/agpl-3.0.html">GNU AGPL License</a> and can be downloaded from the onkeyup=""<a target="_blank" href="https://civihr.org">Project website</a>&nbsp;.
```
Comment (mickadoo) [inline civihr_default_theme/templates/page/page.tpl.php]: > downloaded from the onkeyup

File: civihr_default_theme/templates/page/page.tpl.php (hunk 1 above)
Comment (bmnepali) [reply civihr_default_theme/templates/page/page.tpl.php]: Fixed as discussed with Mike.

**Review by mickadoo — CHANGES_REQUESTED** [review]
Typo after fixing link text

File: civihr_default_theme/templates/page/page.tpl.php (hunk 2 above)
Comment (bmnepali) [reply civihr_default_theme/templates/page/page.tpl.php]: Fixed!

**Review by mickadoo — APPROVED** [review]

**Review by coldrunKacper — APPROVED** [review]

**Review by deb1990 — APPROVED** [review]

**Pull request merged**